### ISSUE TITLE
[DevTools Bug] Cannot add node "1751" because a node with that id is already in the Store.

### ISSUE THREAD

**Issue by regosen** [facebook/react#26787]
### Website or app

local repo

### Repro steps

Loading a React component with the React profiler recording enabled

### How often does this bug happen?

Every time

### DevTools package (automated)

react-devtools-extensions

### DevTools version (automated)

4.27.6-7f8c501f6

### Error message (automated)

Cannot add node "1751" because a node with that id is already in the Store.

### Error call stack (automated)

```text
at chrome-extension://fmkadmapgofadopljbjfkapdkoienihi/build/main.js:28581:41
    at bridge_Bridge.emit (chrome-extension://fmkadmapgofadopljbjfkapdkoienihi/build/main.js:26606:22)
    at chrome-extension://fmkadmapgofadopljbjfkapdkoienihi/build/main.js:26775:14
    at listener (chrome-extension://fmkadmapgofadopljbjfkapdkoienihi/build/main.js:57029:39)
```

### GitHub query string (automated)

```text
https://api.github.com/search/issues?q=Cannot add node  because a node with that id is already in the Store. in:title is:issue is:open is:public label:"Component: Developer Tools" repo:facebook/react
```

**Comment by sandip2224** [comment]
![image](https://user-images.githubusercontent.com/61842142/236860050-c9ba57c1-df7e-4c1b-9934-cf933e9d008c.png)


Same issue here.

Devtools version: 4.27.6 (4/20/2023)

**Comment by FlyOrDie** [comment]
Same issue here. Happens, when adding @react-three/fiber Canvas element. @react-three/fiber version 8.13.0

<img width="682" alt="Снимок экрана 2023-05-10 в 22 31 40" src="https://github.com/facebook/react/assets/5297278/a41364b9-07d9-407f-80ba-0a0158e651d0">

**Comment by apenab** [comment]
Same issue:
![image](https://github.com/facebook/react/assets/22795807/54c7ad62-03b4-46f5-ade9-cb32b6ab2d16)

**Comment by IvanYulin** [comment]
same...
do you all have code for a chrome extension?

**Comment by karin-on** [comment]
same with devtools version: 4.27.7 (5/7/2023)

![Screenshot 2023-05-11 at 12 46 43](https://github.com/facebook/react/assets/40007194/98eb5fff-72ad-4a76-b349-50a6a92011c7)

**Comment by sopowe132** [comment]
I've installed a React profiler via Chrome web store (https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi) which created a second copy of an extension under "From other sources" title in extensions panel, after that I've deleted a copy of an react dev tools that was 'native' to edge, and voila, profiler works.

**Comment by brettgoss** [comment]
@hoxyq I am no longer seeing this error after installing the release you linked in #26797 `(Created from revision 67a05d03e on 5/12/2023)`.

**Comment by hoxyq** [comment]
> @hoxyq I am no longer seeing this error after installing the release you linked in #26797 `(Created from revision 67a05d03e on 5/12/2023)`.

Great, thank you! This fix will be included in v4.27.8 release, which I will publish this week.
